src/mixins/name-request-mixin.ts

```diff
@@ -21,55 +21,57 @@ export default class NameRequestMixin extends Vue {
    * @param nrNumber the name request number to validate
    * @param phone the applicant's phone number
    * @param email the applicant's email address
-   * @returns the name request response payload
+   * @returns the name request object
    */
-  async validateNameRequest (nrNumber: string, phone?: string, email?: string): Promise<NrResponseIF> {
-    const nrResponse: NrResponseIF = await LegalServices.fetchNameRequest(nrNumber, phone, email).catch(error => {
+  async fetchValidateNameRequest (nrNumber: string, phone?: string, email?: string): Promise<NameRequestIF> {
+    const nameRequest = await LegalServices.fetchNameRequest(nrNumber, phone, email).catch(error => {
       if (error?.response?.status === StatusCodes.NOT_FOUND) {
         this.$root.$emit('invalid-name-request', NameRequestStates.NOT_FOUND)
         throw new Error(`${nrNumber} not found`) // Sent invalid NR number
-      } else if (error?.response?.status === StatusCodes.BAD_REQUEST) {
-        this.$root.$emit('invalid-name-request', NameRequestStates.INCORRECT_CONTACT)
+      }
+      if (error?.response?.status === StatusCodes.BAD_REQUEST) {
+        this.$root.$emit('invalid-name-request', 'INCORRECT_CONTACT')
```

Comment:
Temporarily using strings instead of enum here. This will go away when this mixin and its use is refactored in a future ticket.